simulate-travis.py

```diff
@@ -11,14 +11,15 @@
 import subprocess as sp
 import shlex
 import argparse
+import tempfile
 
 if sys.version_info.major == 3:
     PY3 = True
-    from urllib.request import urlretrieve, urlcleanup
+    from urllib.request import urlretrieve
```

Comment:
Definitely. For now though I think it's simple enough to warrant not having six as an additional dep.